e2e/test/scenarios/dashboard-filters-2/dashboard-filters-query-stages.cy.spec.ts

```diff
@@ -138,90 +138,6 @@ describe("scenarios > dashboard > filters > query stages", () => {
     changeSynchronousBatchUpdateSetting(false);
   });
 
-  // Sanity checks. If the base queries tests fail then something is very wrong.
```

Comment:
These "sanity checks" were rather useless.
I decided to remove them to improve performance.